common/src/main/java/com/wynntils/functions/generic/MathFunctions.java

```diff
@@ -12,30 +12,35 @@ public final class MathFunctions {
     public static class AddFunction extends GenericFunction<Double> {
         @Override
         public Double getValue(FunctionArguments arguments) {
-            return arguments.getArgument("first").getDoubleValue()
-                    + arguments.getArgument("second").getDoubleValue();
+            List<Number> values = arguments.getArgument("values").asList().getValues();
+
+            return values.stream().mapToDouble(Number::doubleValue).sum();
         }
 
         @Override
         public FunctionArguments.RequiredArgumentBuilder getRequiredArgumentsBuilder() {
-            return new FunctionArguments.RequiredArgumentBuilder(List.of(
-                    new FunctionArguments.Argument<>("first", Number.class, null),
-                    new FunctionArguments.Argument<>("second", Number.class, null)));
+            return new FunctionArguments.RequiredArgumentBuilder(
+                    List.of(new FunctionArguments.ListArgument("values", Number.class)));
         }
     }
 
     public static class SubtractFunction extends GenericFunction<Double> {
         @Override
         public Double getValue(FunctionArguments arguments) {
-            return arguments.getArgument("first").getDoubleValue()
-                    - arguments.getArgument("second").getDoubleValue();
+            List<Number> values = arguments.getArgument("values").asList().getValues();
+
+            if (values.size() == 1) {
+                return values.get(0).doubleValue();
+            }
+
+            return values.get(0).doubleValue()
```

Comment:
So you are saying A - B - C - D should be interpreted as A - (B + C + D)? It could just as well have been A - (B - (C - D))). Since subtraction is not commutative, it is not clear how to interpret it with multiple operands; this is typically not done and I recommend against it.